mne/epochs.py

```diff
@@ -698,6 +701,16 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         """Compute the mean or std over epochs and return Evoked."""
         _do_std = True if mode == 'stderr' else False
 
+        # if instance contains ICA channels they won't be included unless picks
+        # is specified
+        check_ICA = [x.startswith('ICA') for x in self.ch_names]
```

Comment:
I think you could nest all this under `if picks is None:` - no need to check for ica channels otherwise and you don't need `and picks is None` twice later. But that's a nitpick.